tools/gulp/packaging/inline-resources.ts

```diff
@@ -0,0 +1,58 @@
+/* tslint:disable:no-eval */
+
+import {dirname, join} from 'path';
+import {readFileSync, writeFileSync} from 'fs';
+import {sync as glob} from 'glob';
+
+/** Finds all JavaScript files and inlines all external resources of Angular components. */
+export function inlineResourcesFolder(folderPath: string) {
```

Comment:
I thought about that but `AllResources` sounds like it all resources a given file.